subjects/cybersecurity/hole-in-bin/audit/README.md

```diff
@@ -1,29 +1,51 @@
 #### General
 
-###### Is the student able to explain clearly what reverse engineering means?
+##### Check the Repo content
 
-###### Is the student able to explain clearly what is the difference between decompilation and disassembly and hex-editor?
+- A `README.md` file, Which explains all the steps to bypass all exercises.
+- All used tools and scripts.
 
-###### Is the student able to explain clearly how he patch the program?
+###### Are all the required files present?
 
-###### Is the student able to explain clearly how the "hole-in-bin" program works and how he pass the verification?
+##### Setup the virtual machine
 
-##### Check the Repo content
+1. Download the virtual machine image [hole-in-bin.ova](https://assets.01-edu.org/cybersecurity/hole-in-bin/hole-in-bin.ova). This image contains all the binaries you will need for the audit.
+
+SHA1: 7db09b7a8fdfe25c286561dfa7ca5b50718bd60c
+
+2. Load the virtual machine image into your virtualization software of choice (e.g., VirtualBox, VMWare).
+
+3. Login using the provided credentials (username: user, password: user).
+
+##### Ask the student to disassemble and explain the binaries
+
+> Using a decompiler is forbidden, use a disassembler instead of it!
+
+- The compiler is used to convert high-level programming language code into machine language code.
+- The assembler converts assembly-level language code into machine language code.
+
+###### Did the student capable to disassemble the binaries?
+
+###### Did the student capable to explain the functionality of the all binaries?
```

Comment:
same here, not "Did", and , "functionality of the all binaries?"
should be "functionality of all the binaries?"

Reply:
fixed